grpc.go

```diff
@@ -71,6 +73,17 @@ const onlineCPUMemWaitTime = 100 * time.Millisecond
 
 const onlineCPUMaxTries = 10
 
+const dockerCpusetMode = 0644
+
+// handleError will log the specified error if wait is true
+func handleError(wait bool, err error) error {
```

Comment:
This is different than what is described in the commit message. Could you please split this PR into two separate commits ?

Reply:
@devimc Thanks for splitting into two commits, but two things:
- You should provide an issue ID through `Fixes #XXX` in the new commit message
- Please add, as part of this commit, all the code that has been changed with the introduction of this new function `handleError()`. This way, it's easier to review the commit related to `cpuset` only.